src/views/Home/events-chapel-user-session.json

```diff
@@ -4,7 +4,7 @@
     "CHDate": "2017-09-27T10:25:00",
```

Comment:
Now that we are using live data, we can remove this file and `session-days-left.json`. Can you take care of that?